#!/usr/bin/env python
PACKAGE = "heiman"
#import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("temperatura_mean", double_t, 0, "Media de la temperatura", 32, -100, 200)
gen.add("temperatura_stddev", double_t, 0, "Desviacion estandar de la temperatura", 2, 0, 10)

exit(gen.generate(PACKAGE, "heiman", "getvectorfromHTPApublished"))
